c++/src/capnp/schema.h

```diff
@@ -349,9 +468,18 @@ class StructSchema::Field {
   StructSchema parent;
   uint index;
   schema::Field::Reader proto;
+  Type type;
+  schema::Value::Reader defaultValue;
+
+  static Type resolveType(StructSchema parent, uint index, schema::Field::Reader proto);
```

Comment:
This doesn't need to be static. If it weren't static then you wouldn't need to pass copies of all the members as params.

Reply:
Updated